src/main/java/dev/hephaestus/glowcase/client/gui/screen/ingame/TextBlockEditScreen.java

```diff
@@ -138,19 +138,21 @@ public void render(DrawContext context, int mouseX, int mouseY, float delta) {
 			context.getMatrices().push();
 			context.getMatrices().translate(0, 40 + 2 * this.width / 100F, 0);
 			for (int i = 0; i < this.textBlockEntity.lines.size(); ++i) {
-				int lineWidth = this.textRenderer.getWidth(this.textBlockEntity.lines.get(i));
+				var text = this.currentRow == i ? Text.literal(this.textBlockEntity.getRawLine(i)) : this.textBlockEntity.lines.get(i);
```

Comment:
Just to make sure I'm reading this right - it'll have the raw line for whichever line you've got selected and switch to the formatted version for any non-selected lines? If so, that's *very* nice

Reply:
Yep exactly